kornia/geometry/homography.py

```diff
@@ -1,21 +1,26 @@
+import warnings
+
 import torch
 from typing import Tuple
+from typing import Optional
 
 import kornia
 from kornia.geometry.epipolar import normalize_points
 
 TupleTensor = Tuple[torch.Tensor, torch.Tensor]
 
 
-def find_homography_dlt(points1: torch.Tensor, points2: torch.Tensor, weights: torch.Tensor) -> torch.Tensor:
+def find_homography_dlt(points1: torch.Tensor, points2: torch.Tensor, weights: Optional[torch.Tensor] = None) -> \
```

Comment:
remove backslash and reformat as:

```python
def find_homography_dlt(
    points1: torch.Tensor, points2: torch.Tensor, weights: Optional[torch.Tensor] = None) -> torch.Tensor
```

or

```python
def find_homography_dlt(
    points1: torch.Tensor, points2: torch.Tensor, weights: Optional[torch.Tensor] = None
) -> torch.Tensor
```

or use: `make autopep8` from project root

Reply:
Done